AutolayoutPractice/RemindersList/Views/AddReminderViewController.swift

```diff
@@ -7,8 +7,14 @@
 
 import UIKit
 
+protocol AddReminderDelegate: AnyObject {
+    func addReminder(title: String, description: String, dueDate: Date)
```

Comment:
wanted to get your thoughts on using delegates with MVVM

Reply:
Several approaches:

1. Combine
2. Data Store + State Propagation
3. TCA

Reply:
Combine

var resultsPassthroughSubject = PassthroughSubject<NewReminderData, Never>()
init(resultListener: AnyPublisher<NewReminderData, Never>) {
    resultsListener.subscribe(resultsPassthroughSubject)
}

func saveAction() {
    resultsPassthroughSubject.send(NewReminderData(title: titleTextField.text ?? "", description: descriptionTextView.text ?? "", dueDate: dueDatePicker.date))
}